udata/api/oauth2.py

```diff
@@ -287,8 +317,6 @@ def authorize(*args, **kwargs):
         else:
             grant_user = None
         return oauth.create_authorization_response(grant_user=grant_user)
-    else:
-        abort(405)
```

Comment:
Why remove this? Still seems useful, is someone sends a PUT or whatever

Reply:
I do not see the use of it, as the `methods=['GET', 'POST']` in `@blueprint.route('/authorize', methods=['GET', 'POST'])` should handle it already.